packages/yoastseo/src/worker/AnalysisWebWorker.js

```diff
@@ -1133,10 +1125,10 @@ export default class AnalysisWebWorker {
 			const analysisCombination = {
 				oldAssessor: this._contentAssessor,
 				treeAssessor: this._contentTreeAssessor,
-				scoreAggregator: this._contentScoreAggregator,
 			};
+
 			// Set the locale (we are more lenient for languages that have full analysis support).
-			analysisCombination.scoreAggregator.setLocale( this._configuration.locale );
+			analysisCombination.oldAssessor.getScoreAggregator().setLocale( this._configuration.locale );
```

Comment:
It's not nice that this function is only available to the ReadabilityScoreAggregator. I would consider moving it to the abstract base class.